addons/account/models/account_move.py

```diff
@@ -2944,7 +2944,8 @@ def _prepare_invoice_aggregated_taxes(self, filter_invl_to_apply=None, filter_ta
         # This difference is then distributed evenly across the 'tax_values_list' in 'to_process'
         # such that the manual and computed tax amounts match.
         # The updated tax information is later used by '_aggregate_taxes' to compute the right tax amounts (consistently on all levels).
-        tax_lines = self.line_ids.filtered(lambda x: x.display_type == 'tax')
+        # Skip this step if a tax values filter is used, in order to not override it with the current tax computation
+        tax_lines = self.line_ids.filtered(lambda x: x.display_type == 'tax' and not filter_invl_to_apply)
```

Comment:
This is breaking the account_edi_ubl_cii module

Reply:
This doesn't do any sense to me.
At least it should be using the tax filter (but these filter functions look very weird at first glance)

Reply:
The idea here is: If the `filter_invl_to_apply` has been applied and a base line is excluded from the lines to process, then the tax computation will be different from the invoice, so we should not try to correct the tax amount.
The filter `filter_tax_values_to_apply` will be used later, in `_aggregate_taxes`